docs/data-sources/team.md

```diff
@@ -8,14 +8,21 @@ Buildkite documentation: https://buildkite.com/docs/pipelines/permissions
 ## Example Usage
 
 ```hcl
-data "buildkite_team" "my_team" {
-  slug = "my_team"
+data "buildkite_team" "my_team_data" {
+    depends_on = [buildkite_team.my_team]
```

Comment:
I dont think we should put this in the example. A datasource would be used when the resource itself is not created through terraform
```suggestion
```